packages/design-tokens/design-token-previews/utils/createFontSizePreview.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
```

Comment:
Why do we still need this one? Wouldn't it be possible to use the `createBox`?

Reply:
It's not possible because we have to use the fontSize tokens from the theme in the `Box.div`